core/templates/dev/head/css/oppia.css

```diff
@@ -762,7 +825,7 @@ textarea {
   border-radius: 0;
   margin-top: 10px;
 }
-/* In Bootstrap,white-space for dropdown list is no-wrap.Use this instead to allign ticks and
+/* In Bootstrap,white-space for dropdown list is no-wrap.Use this instead to align ticks and
```

Comment:
nits: add space after ',' and '.'

(Btw, I do realize these are legacy errors. Thanks for fixing the typo!)